pyfr/backends/openmp/blasext.py

```diff
@@ -55,23 +55,31 @@ def errest(self, x, y, z, *, norm):
         if x.traits != y.traits != z.traits:
             raise ValueError('Incompatible matrix types')
 
-        cnt = x.leaddim*x.nrow
-        dtype = x.dtype
+        nrow, ldim, dtype = x.traits
+        ncola, ncolb = x.ioshape[1:]
 
         # Render the reduction kernel template
-        src = self.backend.lookup.get_template('errest').render(norm=norm)
+        src = self.backend.lookup.get_template('errest').render(norm=norm,
+                                                                ncola=ncola)
+
+        # Array for the error estimate
+        error = np.zeros((1, ncola))
```

Comment:
The backend.matrix that we use for the output array buffer wants the second dimension.